tools/commands/build.py

```diff
@@ -31,9 +29,32 @@ def __str__(self):
     def clean(self):
         if os.path.exists(self.build_path):
             logger.info("cleaning {}".format(self.name))
-            os.system("xargs rm < {}/install_manifest.txt".format(self.build_path))
             shutil.rmtree(self.build_path)
 
+    def distclean(self):
+        self.clean()
```

Comment:
This function is not used.

Reply:
What it used to do is delete the whole build directory - `maptools build all -c distclean` == rm -rf build/
also, the positional argument in case of distclean is ignored, since it deletes the whole build folder.
If we move the install folder outside of the build folder, then distclean needs to delete the install folder as well.

I suggest to either remove distclean completely (it is not really distclean per se - see https://www.gnu.org/prep/standards/html_node/Standard-Targets.html) or keep it exactly as it was - simply ignoring the module and deleting everything. I tend to prefer the latter since people are used to distclean as it is now.